packages/devtools_app/lib/src/screens/logging/logging_screen.dart

```diff
@@ -47,7 +45,7 @@ class LoggingScreen extends Screen {
 
     return StreamBuilder<String>(
       initialData: controller.statusText,
-      stream: controller.onLogStatusChanged,
+      stream: controller.onLogStatusChanged as Stream<String>?,
```

Comment:
if you give onLogStatusChanged type `Stream<String>` I think you can remove this cast

Reply:
done